packages/grid/_modules_/grid/hooks/features/columns/useGridColumns.ts

```diff
@@ -145,15 +136,15 @@ export function useGridColumns(
   const allColumns = useGridSelector(apiRef, allGridColumnsSelector);
   const visibleColumns = useGridSelector(apiRef, visibleGridColumnsSelector);
 
-  const updateState = React.useCallback(
-    (newState: GridColumnsState, emit = true) => {
+  const getGridColumnsState = React.useCallback(
```

Comment:
This function doesn't return anything.

```suggestion
  const updateColumnsState = React.useCallback(
```

We could try, in another PR, to remove it and use only `setColumnsState`.

Reply:
Right, I meant to use 

```suggestion
  const setGridColumnsState = React.useCallback(
```